# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/nsNSSModule.cpp
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 security/manager/ssl/nsNSSModule.cpp
--- a/security/manager/ssl/nsNSSModule.cpp	Wed Jun 07 22:44:15 2023 +0000
+++ b/security/manager/ssl/nsNSSModule.cpp	Wed Jun 07 23:03:21 2023 +0000
@@ -15,7 +15,6 @@
 #include "mozilla/ModuleUtils.h"
 #include "mozilla/SyncRunnable.h"
 #include "nsCertTree.h"
-#include "nsCryptoHash.h"
 #include "nsNSSCertificateDB.h"
 #include "nsPK11TokenDB.h"
 #include "nsRandomGenerator.h"
@@ -96,7 +95,6 @@
 IMPL(PKCS11ModuleDB, nullptr)
 IMPL(nsNSSCertificateDB, nullptr)
 IMPL(nsCertTree, nullptr)
-IMPL(nsCryptoHash, nullptr, ProcessRestriction::AnyProcess)
 IMPL(ContentSignatureVerifier, nullptr)
 IMPL(nsRandomGenerator, nullptr, ProcessRestriction::AnyProcess)
 IMPL(OSKeyStore, nullptr, ProcessRestriction::ParentProcessOnly,